App.js

```diff
@@ -1,61 +1,83 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
-import { Provider } from 'react-redux';
+import { Button } from 'react-native';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+import { logoutUser } from './components/Auth/Services/client';
+import {
+  authenticateUser,
+  setUserAuthenticationToken,
+} from './redux/authSlice';
 import { store } from './redux/store';
 
+import HomeScreen from './screens/HomeScreen';
```

Comment:
A general practice with something like this would be to create an index.js file inside of the screens folder. You would import each screen there and then re-export it. This would allow you to do:

`import { HomeScreen, LandingScreen, LoginInScreen, SignUpScreen } from ./screens`